### ISSUE TITLE
Cannot Harvest Corn Properly

### ISSUE THREAD

**Issue by Adnarim7** [Lance5057/ExtraDelight#75]
Upon attempting to harvest corn, I get the kernels directly. From my understanding of the mod it is supposed to give unshucked corn which then gives you corn on the cob and using a knife on the corn on the cob gives the kernels. I cannot figure out why upon harvest, all of these steps are skipped and it just gives straight kernels. I do have another mod that adds corn, called Haunted Harvest, BUT I disabled that mod's corn since I am using this one. Any ideas? I am thinking this might be an incompatibility because I haven't seen this issue mentioned by anyone else. 

I have Quark which slightly alters crop harvesting (makes it so that seeds are automatically replanted upon harvest) however it doesn't mess with the harvesting of other modded crops so I have a hard time believing it is causing this issue with the corn. 

Could this be because the Extra Delight corn is missing expected tags? The following are not present:
forge:vegetables/corn
forge:crops/corn
forge:vegetables

**Comment by Cyndriarr** [comment]
It is an issue with Quark and I figured out a config in quark that fixes it:
1. Open Quark Settings
2. Go to Tweaks Category
3. Scroll down to Sub-Categories and select Simple Harvest
4. Select Right Clickable Blocks
5. Add a new line as follows: extradelight:corn_top
6. Save settings
7. When corn grows, right click the top part with a hoe
8. Have corn and enjoy!

**Comment by Adnarim7** [comment]
Awesome thank you!

**Comment by woshiruozhi233** [comment]
I also encountered similar problems in the game. I modified the configuration of quark according to your method and did not use the harvest crop MOD. However, my corn did not even drop corn kernels after planting. I tried to wait there for it to mature and found that when the planting progress reached 100%, the crop planting progress instantly reached 0% and automatically replanted.（By the way, this corn looks like two different things after it matures...)
Game version:1.19.2
Mod version:extradelight-1.2.2
![11](https://github.com/Lance5057/ExtraDelight/assets/154936854/27df8f63-0ce7-4848-abaa-1ac062125ed8)
![12](https://github.com/Lance5057/ExtraDelight/assets/154936854/b5137983-d5d1-4488-a439-3980f8bd6027)
